baseline.yaml

```
@@ -649,6 +649,30 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+  - id: OSPS-DO-15
+    maturity_level: 2
+    category: Documentation
+    criteria: |
+       The project documentation MUST include a 
+       descriptive statement about how the project
+       selects, obtains, and tracks its dependencies.
+    objective: |
+       Projects need to understand the security qualities
+       of the software components they ingest or leverage 
+       as dependencies.  
+       
+       Having a process and policy for what criteria are 
+       used in this evaluation helps downstream consumers 
+       understand the project's rationale.
+    implementation: |
+       Create a status check that checks the project's
+       version control system for this dependency 
+       ingestion review.
```

```suggestion
       Provide documentation on how dependencies are handled.
       For selection, document what criteria are considered or refer to a document that does so, e.g., the
       <a href="https://best.openssf.org/Concise-Guide-for-Evaluating-Open-Source-Software">Concise Guide for Evaluating Open Source Software</a>.
       If a package manager is used to obtain and track dependencies, note that, otherwise
       explain how dependencies are obtained and tracked.
    autofill:
       Create a status check that checks the project's
       change tracking system (e.g., pull requests or merge requests) for this dependency 
       ingestion review when a new set of dependencies is proposed.
```

The term "version control system" (VCS) doesn't seem correct here. E.g., The tool "git" is the most popular VCS, but it won't have this data. GitHub and GitLab typically do, but they're more than git and provide more than VCS.